source/fuzz/fuzzer_pass_add_global_variables.cpp

```diff
@@ -30,8 +30,20 @@ FuzzerPassAddGlobalVariables::FuzzerPassAddGlobalVariables(
 FuzzerPassAddGlobalVariables::~FuzzerPassAddGlobalVariables() = default;
 
 void FuzzerPassAddGlobalVariables::Apply() {
+  SpvStorageClass variable_storage_class = SpvStorageClassPrivate;
+  for (auto& entry_point : GetIRContext()->module()->entry_points()) {
+    // If the execution model of some entry point is GLCompute,
+    // then the variable storage class may be Workgroup.
+    if (entry_point.GetSingleWordInOperand(0) == SpvExecutionModelGLCompute) {
+      variable_storage_class = GetFuzzerContext()->ChooseEven()
```

Comment:
Instead of hard-coding this as 50:50, could you add a constant to FuzzerContext for the probability of choosing workgroup storage class for a variable?  I'm fine with it being 50:50 by default, but I'd like to give it a name so that we can control it later.

Reply:
Done.